src/js/components/NestedServiceLinks.js

```diff
@@ -175,7 +160,6 @@ const classPropType = PropTypes.oneOfType([
 
 NestedServiceLinks.propTypes = {
```

Comment:
You forgot to add `serviceLink` to propTypes here.

Reply:
good catch